abipy/abio/tests/test_factories.py

```diff
@@ -92,6 +90,61 @@ def test_g0w0_with_ppmodel_inputs(self):
         flow.allocate()
         #flow.make_scheduler().start()
 
+    def test_convergence_inputs_single(self):
+        """Testing g0w0_convergence_input factory single calculation."""
+        scf_kppa, scf_nband, nscf_nband = 10, 10, [10]
+        ecuteps, ecutsigx = [2], 2
+
+        inputs = g0w0_convergence_inputs(self.si_structure, self.si_pseudo, scf_kppa, nscf_nband, ecuteps, ecutsigx,
+                                         ecut_s=[2], scf_nband=scf_nband, ecut=2)
+        # accuracy="normal", spin_mode="polarized", smearing="fermi_dirac:0.1 eV",
+        # ppmodel="godby", charge=0.0, scf_algorithm=None, inclvkb=2, scr_nband=None,
+        # sigma_nband=None, gw_qprange=1):
+
+        # one scf, one nscf and one screening / sigma multi
+        self.assertEqual(len(inputs), 4)
+        self.assertIsInstance(inputs[0][0], AbinitInput)
+        self.assertIsInstance(inputs[1][0], AbinitInput)
+        self.assertIsInstance(inputs[2][0], AbinitInput)
+        self.assertIsInstance(inputs[3][0], AbinitInput)
+
+        self.assertEqual(inputs[0][0].variable_checksum(), 8922380746077674361)
+        self.assertEqual(inputs[1][0].variable_checksum(), -6328062324283394468)
+        self.assertEqual(inputs[2][0].variable_checksum(), -4406035156560398972)
+        self.assertEqual(inputs[3][0].variable_checksum(), 7970609742942507895)
+
+#        for input in [inputs[0][0], inputs[1][0], inputs[2][0], inputs[3][0]]:
+
+        for input in [item for sublist in inputs for item in sublist]:
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'input'](https://www.codacy.com/app/gmatteo/abipy/file/2694250468/issues/source?bid=3176346&fileBranchId=3340437#l118)